rpi-convert-stage-5.sh

```diff
@@ -66,8 +66,16 @@ build_uboot_files() {
 	fdt addr ${fdt_addr} && fdt get value bootargs /chosen bootargs
 	run mender_setup
 	mmc dev ${mender_uboot_dev}
-	load ${mender_uboot_root} ${kernel_addr_r} /boot/zImage
-	bootz ${kernel_addr_r} - ${fdt_addr}
+	if [ -f /boot/zImage ]; then
+	  load ${mender_uboot_root} ${kernel_addr_r} /boot/zImage
+	  bootz ${kernel_addr_r} - ${fdt_addr}
+	elif [ -f /boot/uImage ]; then
+	  load ${mender_uboot_root} ${kernel_addr_r} /boot/uImage
+	  bootm ${kernel_addr_r} - ${fdt_addr}
+	else
+	  echo "No bootable Kernel found. Aborting."
+	  exit 1
```

Comment:
Do not exit the script here, as it should be possible to reach `mender_try_to_recover` to detect issues caused by an update.